### ISSUE TITLE
Empty workspaces parsed for packages

### ISSUE THREAD

**Issue by itsezc** [oven-sh/bun#2456]
### What version of Bun is running?

0.5.8

### What platform is your computer?

WSL / Windows

### What steps can reproduce the bug?

Create a package.json with workspaces, and add an entry with a glob of an empty dir i.e. `empty_projects/*` and run `bun install` you will get the following error:

```
warn: workspaces directory prefix not found "empty_projects/"
```

### What is the expected behavior?

Similar to NPM, you would expect bun to ignore this glob pattern

**Comment by zaida04** [comment]
Bumping this

**Comment by dylan-conway** [comment]
fixed in #10462
